source/js/components/people-box/people-box.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
I thought about just adding code to `people.jsx` and `person.jsx` but since there are some complicated logics going on there I chose to make a separate component `PeopleBox` instead.